data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1473,45 +1496,80 @@ descriptions:new_tribe {
+            -- TRANSLATORS: Note helptext for an atlantean production site: Gold Spinning Mill
+            note = pgettext("atlanteans_building", "Do not build until you have a steady gold supply."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site: Gold Spinning Mill
+            lore = pgettext("atlanteans_building", "No this is not Rumpelstiltskin spinning gold from straw, we need real gold"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site: Gold Spinning Mill
+            lore_author = pgettext("atlanteans_building", "Painted on a Gold Spinning Mill wihout supplies.")
          }
       },
       {
          name = "atlanteans_scouts_house",
          helptexts = {
+            -- TRANSLATORS: Special helptext for an atlantean production site: Scout's House without road connection
             no_scouting_building_connected = pgettext("atlanteans_building", "You need to connect this flag to a scout’s house before you can send a scout here."),
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Scout's House
-            purpose = pgettext("building", "Explores unknown territory.")
+            purpose = pgettext("building", "Explores unknown territory."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Scout's House
+            note = pgettext("atlanteans_building", "Will need fish an bread to have his meal the Atlantean way"),
+            -- TRANSLATORS: Lore helptext for an atlantean production site:  Scout's House
+            lore = pgettext("atlanteans_building", "You do not see everything from you high towers. Sometimes you need someone to find out in person, thats me!"),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site:  Scout's House
+            lore_author = pgettext("atlanteans_building", "A Scout answering a soldier")
          }
       },
 
-      -- Medium
+      -- Medium sized Buildings
       {
          name = "atlanteans_sawmill",
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an atlantean production site: Sawmill
-            purpose = pgettext("building", "Saws logs to produce planks.")
+            purpose = pgettext("building", "Saws logs to produce planks."),
+            -- TRANSLATORS: Note helptext for an atlantean production site: Sawmill
+            note = pgettext("atlanteans_building", "Will need at least four Woodcutters for a constant supply of planks."),
+            -- TRANSLATORS: Lore helptext for an atlantean production site:  Sawmill
+            lore = pgettext("atlanteans_building", "This Vertical Axis design makes operation simple, yet powerfull. Patent pending."),
+            -- TRANSLATORS: Lore author helptext for an atlantean production site:  Sawmill
+            lore_author = pgettext("atlanteans_building", "The Tom Sawyer answering some imperial dude")
```

Comment:
Atlanteans are respectful, "some dude" doesn't sound right IMHO

Reply:
"... an imperial visitor" , OTOH Atlanteans do no think very high about the other tribes.